### PR TITLE
Add a sentence stating not to sign in with Twitter on 5/9

### LINKED ISSUE #14383: Update Gitter challenge copy to make it clear campers should sign in with GitHub (and not Twitter)
#### Challenge Name
https://www.freecodecamp.com/challenges/create-a-github-account-and-join-our-chat-rooms

#### Issue Description
Gitter recently introduced Twitter authentication. In our Gitter rooms, we force campers to be signed into Gitter with GitHub (and not Twitter). So we need to update the copy in this challenge to clarify that they should use GitHub - and not Twitter - to create their Gitter account.

Here's the current copy:

create_a_github_account_and_join_our_chat_rooms___free_code_camp

This challenge is located within a JSON file within /seed/challenges

### PR DESCRIPTION
**Pull request by MeamonB**
#### Pre-Submission Checklist
- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change
- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:
- [x] Tested changes locally.
- [x] Closes currently open issue: Closes #14383 

#### Description
In issue #14383, @QuincyLarson asked so that on 5/9 of 00-getting-started, it specifically states to not sign in with Twitter. I added that statement in, and translated it into some of the languages.

### CODE REVIEW

File: seed/challenges/00-getting-started/getting-started.json [hunk 1]
```diff
@@ -781,7 +781,7 @@
             [
               "//i.imgur.com/OmRmLB4.gif",
               "Un gif mostrándote como pulsar en el enlace inferior para ir a nuestra sala de chat y pulsar en el botón de \"iniciar sesión con GitHub\". Entonces puedes pulsar en el próximo campo de texto y escribir un mensaje a tus compañeros campistas.",
-              "Ahora que tienes una cuenta de GitHub, puedes unirte a nuestro chat principal al iniciar sesión con GitHub. Preséntate diciendo \"Hello world!\" Cuéntales a los tus compañeros campistas como encontraste freeCodeCamp. Además cuéntanos porque deseas aprender a programar.",
+              "Ahora que tienes una cuenta de GitHub, puedes unirte a nuestro chat principal al iniciar sesión con GitHub. No ingrese con Twitter. Preséntate diciendo \"Hello world!\" Cuéntales a los tus compañeros campistas como encontraste freeCodeCamp. Además cuéntanos porque deseas aprender a programar.",
```
Comment (dhcodes) [inline seed/challenges/00-getting-started/getting-started.json]: please add "por favor" to this translation.

**Comment by dhcodes** [comment]
To make the update, if you still have the branch on your computer, make the change, add the file with `git add [filename]`. Do a `git commit --amend --no-edit`. Then `git push origin [your-branch-name] -f`.

Thanks @MeamonB

File: seed/challenges/00-getting-started/getting-started.json (hunk 1 above)
Comment (MeamonB) [inline seed/challenges/00-getting-started/getting-started.json]: Added it!

**Comment by dhcodes** [comment]
Tested locally and **LGTM**. Congratulations on your first contribution to freeCodeCamp @MeamonB ! Looking forward to many more :) 👍 💯 🌮 🦅

**Pull request merged**